packages/playwright/src/errorContext.ts

```diff
@@ -139,12 +140,39 @@ export async function attachErrorContext(testInfo: TestInfoImpl, format: 'markdo
   }
 }
 
-async function loadSource(file: string, sourceCache: Map<string, string>) {
+type Location = {
```

Comment:
I am pretty sure we already have a `Location` type 😄

Reply:
There is `Location` in html-reporter and `SourceLocation` in trace-viewer. Both seem wrong. Also `StackFrame` in isomorphic but probably fine to use my own?